integration-tests/test/basic-l1-l2-communication.spec.ts

```diff
@@ -116,7 +115,8 @@ describe.skip('Basic L1<>L2 Communication', async () => {
       expect(remoteReceipt.status).to.equal(1)
     })
 
-    it('should have a receipt with a status of 0 for a failed message', async () => {
+    // SKIP: until we decide what should be done in this case
+    it.skip('should have a receipt with a status of 0 for a failed message', async () => {
```

Comment:
This is pending a conversation we're having about how this should be handled. So far I think the best option is to make the OVM_L2CrossDomainMessenger revert if the sub-call fails.